docs/howto/fiber_dump.md

```diff
@@ -0,0 +1,95 @@
+---
+id: howto_fiber_dump
+title:  "Sending Interrupt Signal to get fiber dump"
+---
+
+You can send a `SIGUSR2` signal from your os to interrupt your app and a get a fiber dump of all fibers.
```

Comment:
Great documentation! Is there a way to get this to work when you do CTRL-C in SBT? That would be a really nice feature. Also may be good to mention current lack of support on Scala JS and Scala Native.